packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -58,13 +64,71 @@ export async function fetchSnapshot(
     );
 }
 
+export async function redeemSharingLink(
```

Comment:
I think there is no need to export this function as it's used only in this file. Can you please check other places?